pdns/lua-auth4.cc

```diff
@@ -171,4 +171,25 @@ std::unique_ptr<DNSPacket> AuthLua4::prequery(const DNSPacket& q) {
   return nullptr;
 }
 
+bool AuthLua4::axfr_end(const DNSName& zone) {
+  luacall_axfr_end_t::result_type rcode;
+
+  if (d_axfr_end == NULL) return false;
+
+  rcode = d_axfr_end(zone);
+
+  if (rcode < 0) {
+    // failed to execute the AFXR end notification
+    return false;
+  }
+  else if (rcode >= 0) {
+    // success
+    return true;
+  }
+  else
+    throw PDNSException("Cannot understand return code "+std::to_string(rcode)+" in axfr end response");
+
+  return true;
```

Comment:
```suggestion
  return true;
```

Reply:
@pieterlexis: thanks for the suggestion. https://github.com/m0rcq/pdns/commit/92650b0e83e556ea85776a5a55422d5dc15f31a2 addresses this.